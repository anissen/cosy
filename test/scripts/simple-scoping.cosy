let x = 3
{
  print x
  let x = 7
  print x
}
