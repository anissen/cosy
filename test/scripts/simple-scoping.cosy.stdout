â–  test/scripts/simple-scoping.cosy, line 4:
2 | {
3 |   print x
4 |   let x = 7
          ^ Shadows existing variable.
5 |   print x
6 | }

